src/nvim/normal.c

```diff
@@ -4660,7 +4660,12 @@ static void nv_exmode(cmdarg_T *cap)
   if (VIsual_active) {
```

Comment:
Maybe `Q` should be allowed in Visual mode, since both `q` and `@` are allowed.

Reply:
Indeed, it would be great to have that too, working on it right now, and adding tests for it.

Reply:
Both issues should be addressed now.